### PR TITLE
added copy button with toast

### PR DESCRIPTION
**Pull request by imlakshaychauhan**
Added a copy button, which copies the commands on clicking this button and shows a toast after copying.
Here are some screenshots for the same:
Screenshot (4)
Screenshot (3)

### CODE REVIEW

**Comment by devvsakib** [comment]
there are some issues when you copy the solution. Less than sign is also included to clipboard
image

**Review by devvsakib — CHANGES_REQUESTED** [review]
Copied text contain < sign
git pull<git push origin [branch name]

**Comment by devvsakib** [comment]
Everything is perfect and love your work. But we have to remove < sign from clipboard

**Comment by imlakshaychauhan** [comment]
just removing < will work, right?

**Comment by imlakshaychauhan** [comment]
git pull git push origin [branch name]

File: src/components/Error/ErrorCard.jsx [hunk 1]
```diff
@@ -65,24 +68,39 @@ function ErrorCard({ error }) {
...
-            <span className="text-xs">Solution</span>
-            <MdOutlineArrowRightAlt className="text-lg" />
-          </>
-        )}
-      </button>
+          </button>
+          <button
+            className="flex mt-8 items-center gap-2 mx-4 px-3 py-2 border border-gray rounded-lg hover:border-primary hover:text-primary"
+            onClick={() => {
+              navigator.clipboard.writeText(error.solutions);
```
Comment (devvsakib) [inline src/components/Error/ErrorCard.jsx:83]: Use this code in writeText function to remove < sign

```
error.solutions.split("<").join(" ")
```

**Review by devvsakib — CHANGES_REQUESTED** [review]

**Review by devvsakib — APPROVED** [review]
Perfect. Thanks for your contributions🟩

**Pull request merged**